pkg/kev/config/labels.go

```diff
@@ -85,6 +85,9 @@ const (
 	// LabelWorkloadLivenessProbeHTTPPath defines the path for the http liveness probe.
 	LabelWorkloadLivenessProbeHTTPPath = LabelPrefix + "workload.liveness-probe.http-path"
 
+	// LabelWorkloadLivenessProbeTCPPort defines the path for the http liveness probe.
+	LabelWorkloadLivenessProbeTCPPort = LabelPrefix + "workload.liveness-probe.tcp-port"
```

Comment:
Oh, I thought we changed those to be hyphenated. See my previous comment: https://github.com/appvia/kev/pull/399/files#r593321941 

I think we should stick to hyphenated format for consistency for the time being.

Reply:
it's hyphenated now on the "sections" "liveness-probe" "tcp-port", did you mean `workload-liveness-probe-tcp-port` or `workload.liveness-probe-tcp-port`?

Reply:
yeh, the latter, so `workload.liveness-probe-tcp-port`. Sorry should have been more explicit :)